doc/tutorials/installation.rst

```diff
@@ -135,13 +133,13 @@ Installing GDCM
 `GDCM <http://gdcm.sourceforge.net/>`_ is a C++ library for working with
 DICOM datasets that can decompress JPEG, JPEG-LS and JPEG 2000 images.
 
-The wheels on `PyPI <https://pypi.org/project/python-gdcm/>`_ are built by the
+The wheels on `PyPI <https://pypi.org/project/python-gdcm/>`__ are built by the
```

Comment:
Why the double underscore in links, but not in others?

Reply:
If you use the same link text in multiple links (PyPI in this case) with a single underscore you have two named references that use the same name (PyPI), whereas using a double underscore makes it an anonymous reference and the same link text can be reused without any warnings.

Reply:
Ah ok, didn't know that!